[1] [CITE@en[Correctly rethrow exceptions when using Call()]]
([[Ms2ger]]著, [TIME[2019-01-30 02:40:10 +09:00]])
<https://github.com/heycam/webidl/commit/323cf3f2f2a9df8376b44239925d3486121a3398>

[2] [CITE@en[Fix error handling when using Call() · Issue #207 · heycam/webidl]]
([TIME[2019-06-20 13:16:57 +09:00]])
<https://github.com/heycam/webidl/issues/207>

[3] [CITE@en[Fix error handling around Call(). by Ms2ger · Pull Request #618 · heycam/webidl]]
([TIME[2019-06-20 13:18:37 +09:00]])
<https://github.com/heycam/webidl/pull/618>